src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] list = new Figure [6];
+        FigureSupplier figureSupplier = new FigureSupplier();
+        for (int i = 0; i <= 2; i++) {
+            list[i] = figureSupplier.getRandomFigure();
+            System.out.println(list[i].toString());
+        }
+        for (int i = 3; i <= 5; i++) {
+            list[i] = figureSupplier.getDefaultFigure();
+            System.out.println(list[i].toString());

```

Comment:
Similarly to the previous comment, there is no need to call `toString()` explicitly here. You can just use `System.out.println(list[i]);`.